src/main/java/smile/identity/core/models/PackageInformation.java

```diff
@@ -0,0 +1,25 @@
+package smile.identity.core.models;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Stream;
+
+@Getter @Setter @NoArgsConstructor
+public class PackageInformation {
+
+    private final Map<String, Integer> apiVersion = setApiVersion();
+    private final String language = "java";
+
+
+    private Map<String, Integer> setApiVersion(){
+        Map<String, Integer> map = new HashMap<>();
```

Comment:
I'm pretty sure this is supposed to be based on the version of this library.. if anyone has any suggestions on where we can store this information and reuse throughout the code base, let me know. the version is also in the `build.gradle` file and `PreUploadRequest`. Would be nice to have it in one location and refer to it throughout.

Reply:
I'd suggest adding some small function to the build.gradle to output the version to a version.properties file that will then be in the class path for the Java, so it can be easily read in and consumed...

Some example ideas on this method here:
https://stackoverflow.com/questions/33020069/how-to-get-version-attribute-from-a-gradle-build-to-be-included-in-runtime-swing 
https://stackoverflow.com/questions/50117966/get-version-in-kotlin-java-code-in-gradle-project

Reply:
no. this is sadly not for the version of this library. 

it is used by the R in RSD here: https://github.com/smileidentity/Riku/blob/master/Riku/Riku.cpp#L927-L954, for a few things including "Source" attribution here: https://github.com/smileidentity/Riku/blob/master/Riku/Riku.cpp#L1088-L1154.

p.s.: this was one of the first noteworthy bugs I introduced. partly why I remember.

Reply:
@beastawakens sweeeet! I like this 

@tamssokari  so what should this be then? I'm seeing 0.2.0 in the ruby sdk, so im guessing its safe to go with that

Reply:
yes it is.